R/utils.R

```diff
@@ -200,27 +200,128 @@ getRasterLayerNames <- function(startDateTime, endDateTime, by, variables,
   return(layerNames)
 }
 
-#' Get all active FMI weather stations.
+# function fmi_station()
+
+# Reading the local version included within the package is separated into its
+# own function, since it's also used for tests.
+.fmi_stations_local <- function() {
+  system.file("extdata", "fmi_stations.csv", package="fmi") %>%
+    utils::read.table(header=TRUE, sep=";", as.is=TRUE) %>%
```

Comment:
Very minor thing, but since [the English version of the table](http://en.ilmatieteenlaitos.fi/observation-stations) actually uses `.` instead of `,` as decimal mark, I would recommend saving the table using `sep = ","` i.e. a proper CSV.